index.js

```diff
@@ -20,3 +20,16 @@ exports.renderFileAsync = function (filename, options, locals) {
     });
   });
 };
+
+exports.renderAsync = function (str, options, locals) {
+  return new Promise(function (resolve, reject) {
+    var browserifyString = require('browserify-string');
```

Comment:
@TimothyGu This is better.... It will only use `browserify-string` if it's needed.